code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -10,9 +10,16 @@
 
 	health = maxHealth - getBrainLoss()
 
-	//TODO: fix husking
-	if(((maxHealth - getFireLoss()) < config.health_threshold_dead) && stat == DEAD)
-		ChangeToHusk()
+	if(stat == DEAD)
+		if(getFireLoss() > maxHealth * 1.5)
+			ChangeToHusk()
+		if(getFireLoss() > maxHealth * 2.5)
+			ChangeToSkeleton()
+		if(getFireLoss() > maxHealth * 3)
+			for(var/obj/item/I in src)
```

Comment:
This might cause organs and other stuff to drop from the mob.

Reply:
it doesn't